addons/mail/static/src/components/drop_zone/drop_zone.xml

```diff
@@ -2,7 +2,7 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.DropZone" owl="1">
-        <div class="o_DropZone" t-att-class="{ 'o-dragging-inside': state.isDraggingInside }" t-attf-class="{{ className }}" t-on-dragenter="_onDragenter" t-on-dragleave="_onDragleave" t-on-dragover="_onDragover" t-on-drop="_onDrop" t-ref="root">
+        <div class="o_DropZone position-absolute top-0 start-0 align-items-center justify-content-center d-flex w-100 h-100 border-primary bg-white-75 text-primary" t-att-class="{ 'o-dragging-inside': state.isDraggingInside }" t-attf-class="{{ className }}" t-on-dragenter="_onDragenter" t-on-dragleave="_onDragleave" t-on-dragover="_onDragover" t-on-drop="_onDrop" t-ref="root">
```

Comment:
Opacity changed from `90%` to `75%`, but it's not a big deal, as long as opacity is more than `25%` and less than `100%`, in-between value is more about UI consistency and/or preference.